Assets/_Functionalities/ObjectInformation/Scripts/BagInspector.cs

```diff
@@ -145,10 +146,21 @@ private void FindObjectMapping()
                 //lets order all mappings by layerorder (rootindex) from layerdata
                 if (featureSelector.HasFeatureMapping)
 				{
+					List<Feature> filterDuplicates = new List<Feature>();
 					foreach (FeatureMapping feature in featureSelector.FeatureMappings)
 					{
-						if(feature.VisualisationParent.LayerData.ActiveInHierarchy)
+						if (feature.VisualisationParent.LayerData.ActiveInHierarchy)
+						{
+							if(filterDuplicateFeatures)
```

Comment:
kan deze if niet samengevoegd worden met die erboven? ik zou ook filterDuplicateFeatures eerst doen, omdat dat minder zwaar is dan die andere check, dus als filterDuplicateFeatures false is dan hoef je die andere call niet te doen